web/modules/custom/dbcdk_community/tests/src/Unit/Content/FlaggableContentRepositoryTest.php

```diff
@@ -145,13 +153,50 @@ public function unreadWithFlagsDataProvider() {
   }
 
   /**
+   * Test that collections can be fetched by the repository.
+   */
+  public function testContentWithCollections() {
+    // Data.
+    $post = new Post();
+    $post->setId(1);
+    $image_collection = new ImageCollection();
+    $image_collection->setId(1);
+    $image_collection->setPostImageCollection($post->getId());
+    $video_collection = new VideoCollection();
+    $video_collection->setId(2);
+    $video_collection->setPostVideoCollection($post->getId());
+
+    // Stubs.
+    $flag_api_stub = $this->getMock('DBCDK\CommunityServices\Api\FlagApi');
+    $post_api_stub = $this->getMock('DBCDK\CommunityServices\Api\PostApi');
+    $comment_api_stub = $this->getMock('DBCDK\CommunityServices\Api\CommentApi');
+    $post_api_stub->method('postPrototypeGetImage')->willReturn([$image_collection]);
+    $post_api_stub->method('postPrototypeGetVideo')->willReturn([$video_collection]);
```

Comment:
Consider checking that the stub is called with the expected argument.

One way to check this is using [`willReturnMap()`](https://github.com/DBCDK/biblo-admin/blob/feature/refactor-profiles/web/modules/custom/dbcdk_community/tests/src/Unit/Profile/ProfileRepositoryTest.php#L66-L68). Note that the argument to this method is an array of all mandatory and optional arguments and the return value.